lib/paper_trail/model_config.rb

```diff
@@ -145,9 +145,14 @@ def cannot_record_after_destroy?
     #
     # @api private
     def check_presence_of_item_subtype_column(options)
-      return unless options.key?(:limit)
-      return if version_class.item_subtype_column_present?
-      raise format(E_MODEL_LIMIT_REQUIRES_ITEM_SUBTYPE, @model_class.name)
+      begin
+        return unless options.key?(:limit)
+        return if version_class.item_subtype_column_present?
+        raise format(E_MODEL_LIMIT_REQUIRES_ITEM_SUBTYPE, @model_class.name)
+      rescue ActiveRecord::NoDatabaseError => e
```

Comment:
call to `column_names` in `item_subtype_column_present` raises `ActiveRecord::NoDatabaseError` if the database does not exist.